engine/src/main/java/org/hibernate/validator/HibernateValidatorContext.java

```diff
@@ -68,20 +67,6 @@
 	HibernateValidatorContext addValidationValueHandler(ValidatedValueUnwrapper<?> handler);
 
 	/**
-	 * Registers the given time provider with the bootstrapped validator. This provider will be used to obtain the
-	 * current time when validating {@code @Future} and {@code @Past} constraints. If not set or if {@code null} is
-	 * passed as a parameter, the time provider of the {@link javax.validation.ValidatorFactory} is used.
-	 *
-	 * @param timeProvider the time provider to register.
-	 *
-	 * @return {@code this} following the chaining method pattern
-	 *
-	 * @hv.experimental This API is considered experimental and may change in future revisions
```

Comment:
Yes, we discussed about it a few weeks ago on HipChat and we decided to remove it as:
- it was experimental
- having both ClockProvider and TimeProvider would be confusing